bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/photon_proton_NLO_MG_pp_13TeV/TTbar_photon_proton_NLO_MG_pp_13TeV/photo_NLO_M_ttbar_pp_13TeV.txt

```diff
@@ -0,0 +1,10 @@
+#import model sm-no_b_mass
+generate p a > t t~ [QCD]
+set automatic_html_opening False
+output jam_lpp2_2_photo_NLO_ttbar_pp_13TeV
+launch
+launch
+set lpp2 2
```

Comment:
is this for the madstr plugin? please specify in the run_card.dat

Reply:
ttbar does not need madstr pluggin, it can just be applied to tW